src/reference/asciidoc/jdbc.adoc

```diff
@@ -433,6 +433,8 @@ For this purpose, the `INT_CHANNEL_MESSAGE` has a `MESSAGE_PRIORITY` column to s
 In addition, a new `MESSAGE_SEQUENCE` column is also provided to achieve a robust first-in-first-out (FIFO) polling mechanism, even when multiple messages are stored with the same priority in the same millisecond.
 Messages are polled (selected) from the database with `order by MESSAGE_PRIORITY DESC NULLS LAST, CREATED_DATE, MESSAGE_SEQUENCE`.
 
+IMPORTANT: The `capacity` attribute is not allowed when the `message-store` attribute is used.
```

Comment:
Ok, removed that line from `jdbc.adoc`. 

About 

```
 [[channel-configuration-prioritychannel]]
 ===== PriorityChannel Configuration
```

I can't put the line 

```
IMPORTANT: The `capacity` attribute is not allowed when the `message-store` attribute is used.
``` 

it because that section for Priority Channel does not cover nothing about `message-store`, so that is confuse, but just below in the following paragraph I have added (already pushed)

```
Since _version 4.0_, the `priority-channel` child element supports the `message-store` option (`comparator` and `capacity` are not allowed in that case).
```

Practically:
```
(`comparator` and `capacity` are not allowed in that case)
```